### PR TITLE
Add Veganism

### PR DESCRIPTION
**Pull request by sdassow**
https://github.com/sdassow/awesome-veganism

This list contains entries to make veganism easy and accessible to everyone who is interested. There are many rather random, unmoderated, or unmaintained lists created by individuals or companies, with sometimes questionable quality. This list is an attempt to create a community driven approach to guarantee no-nonsense content, 100% aligned with the life-style, in order to minimise the effort needed for an awesome experience.

 - The list is around for 2+ months.
 - Checked with awesome-lint.
 - Reviewed PRs: https://github.com/sindresorhus/awesome/pull/1876#issuecomment-706613089, https://github.com/sindresorhus/awesome/pull/1879#issuecomment-706614447, https://github.com/sindresorhus/awesome/pull/1884, and merged PRs to see what success looks like.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sdassow** [comment]
unicorn

**Comment by sdassow** [comment]
By the way, there is a logo which can be seen on the website (done via stylesheet): https://awesome-veganism.com/
And for anyone interested, the [website source](https://github.com/sdassow/awesome-veganism-website) and the [markdown generator](https://github.com/sdassow/markdown-webgen) related to this project are also open source :-).

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by sdassow** [comment]
Since I've been through the list probably 10+ times by now, I do need some pointers to fix anything that's left :-).

**Comment by sdassow** [comment]
Oh my... how did I miss the PR title... my bad, fixed.

File: readme.md [hunk 1]
```diff
@@ -708,6 +708,7 @@
 - [Biomedical Information Extraction](https://github.com/caufieldjh/awesome-bioie#readme) - How to extract information from unstructured biomedical data and text.
 - [Web Archiving](https://github.com/iipc/awesome-web-archiving#readme) - An effort to preserve the Web for future generations.
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
+- [Veganism](https://github.com/sdassow/awesome-veganism#readme) - Resources for a fun and smooth vegan experience.
```
Comment (oncleben31) [inline readme.md:711]: `Resources for ....` is not compliant with description requirements.

**Review by oncleben31 — CHANGES_REQUESTED** [review]

**Comment by programmin1** [comment]
Interesting list but the restaurants and bakeries list is not yet large enough to be very useful for everyone.

**Comment by neiesc** [comment]
I'm vegan in 4 years, it's awesome, but How you organize per country/city maybe group by them.

File: readme.md (hunk 1 above)
Comment (sdassow) [reply readme.md]: Right. Changed it in 808e99685a01ec9cbe216956fa3c1e1f3a639f56, better?

Comment (oncleben31) [reply readme.md]: Yes. Thank you.

**Comment by Tintwo** [comment]
Hello folks,
I'm just wondering about this PR for this new awesome-list.
I know this actual awesome-list is not closed to any kind of topic. I notice also that for now, the actual list concerns about tech/science/code domains. Your PR seems to concern about society et lifestyle, right ?
So I'm just wondering if it's better to split this awesome list in 2 part, for example as "Sciences&Tech" and "Society", than to have one mega list of all domains which will be harder to maintain and read over time.

Regards.

**Comment by sdassow** [comment]
> Hello folks,
> I'm just wondering about this PR for this new awesome-list.
> I know this actual awesome-list is not closed to any kind of topic. I notice also that for now, the actual list concerns about tech/science/code domains. Your PR seems to concern about society et lifestyle, right ?
> So I'm just wondering if it's better to split this awesome list in 2 part, for example as "Sciences&Tech" and "Society", than to have one mega list of all domains which will be harder to maintain and read over time.
> 
> Regards.

Right, that is a good point. Maybe a separate section? There are some other entries in the "Miscellaneous" section that could qualify for a new "Lifestyle" section as well, like for example "awesome-food", "awesome-mental-health", and "awesome-earth".

**Comment by niedev** [comment]
For me this awesome respects all the guidelines, also verified with awesome-lint.
My girlfriend is vegan so from personal experience I can say that it is a sector in which many interesting innovations are often released regarding products, recipes, restaurants etc. Consequently an awesome list on this topic I think can be very useful.

File: readme.md [hunk 2]
```diff
@@ -715,6 +715,7 @@
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
 - [Credit Modeling](https://github.com/mourarthur/awesome-credit-modeling#readme) - Methods for classifying credit applicants into risk classes.
 - [Ansible](https://github.com/KeyboardInterrupt/awesome-ansible#readme) - A Python-based, open source IT configuration management and automation platform.
+- [Veganism](https://github.com/sdassow/awesome-veganism#readme) - Making veganism easy and accessible to everyone who is interested.
```
Comment (sindresorhus) [inline readme.md:718]: The description should describe **what** veganism is, per guidelines.

Comment (sindresorhus) [reply readme.md:718]: Same with the description in your Awesome list.

**Comment by sindresorhus** [comment]
I would have like to see more blog posts / articles / videos/ etc about veganism, not just vegan items.

**Comment by sdassow** [comment]
Thanks for the feedback! Made corresponding changes, and started adding more informational resources as requested in the last comment, and more will follow.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1348899373795528705

**Pull request merged**